### DISCUSSION TITLE
Documentation Town Hall

### DISCUSSION THREAD

**Discussion by BoDonkey** [apostrophecms/apostrophe#3957] [Announcements]
Howdy!
We have been working hard to update our documentation by covering features and modules that were missing in our current documentation. Moving forward, we are striving to make our documentation updates and additions more responsive to the community's needs. We are experimenting with starting an ongoing series of town halls to outline what we have planned and solicit feedback. This could potentially be expanded to other town halls centered around new feature development and "ask-me-anything" technical discussions.
The first of these is being planned for the third week of December. The topics will be the current state of the documentation and, centrally, our plans for releasing a series of written and video tutorials for onboarding newer developers into the A3 ecosystem. We really would love to have the community's feedback to make sure these documents are valuable and cover pain points that people are experiencing.
Given that Apostrophe is being used worldwide, we know there is no perfect time for scheduling this town hall. To maximize potential attendance, we have made up a Google form to submit when you would prefer we have this meeting. You can vote anonymously and for as many different times as you would like. The link is here: https://forms.gle/4NueSb5rXcgadHhV8
Thanks!
BoDonkey (@BobM)

**Comment by ChineserDragon** [comment]
你好，你的来信已收到，我将尽快回复你。
